extract_sets/training_sets.py

```diff
@@ -0,0 +1,398 @@
+# -*- coding: utf-8 -*-
+"""Generate TrainingSets Sets"""
+
+import abc
+import os
+from pathlib import (
+    Path
+)
+from functools import reduce
+import sys
+
+import exifread
+import lxml.etree as etree
+import numpy as np
```

Comment:
```suggestion
```

Reply:
This fixes a warning from LGTM.

Reply:
Sorry for the irritation, this was rather a WIP commit